embed/config.go

```diff
@@ -408,19 +407,6 @@ func NewConfig() *Config {
 	return cfg
 }
 
-func logTLSHandshakeFailure(conn *tls.Conn, err error) {
```

Comment:
Note for reviewer: the function actual in use is defined here: https://github.com/etcd-io/etcd/blob/7395ed8e5d33c519d1d0c0bf556f7363c3e02894/embed/config_logging.go#L167